etebase_server/settings.py

```diff
@@ -138,6 +143,8 @@
     "/etc/etebase-server/etebase-server.ini",
 ]
 
+ETEBASE_CREATE_USER_FUNC = "django_etebase.utils.create_user_blocked"
```

Comment:
I guess this is also a merge issue because it was just moved to a different place in your commit.